apps/pie-docs/src/foundations/design-tokens/updates-and-maintenance.md

```diff
@@ -145,4 +145,4 @@ Once a new version of the JSON has been published the Android core team is notif
 
 These assets will then be copied into the Android UI repository via a PR. Once the PR is approved and merged new versions of the pie-view and pie compose libraries will be published.
 
-Android applications that required the new tokens will then have their versions of the pie libs updated via another PR. Once this PR is merged the next released version of the application will contain the design token updates.
+Android applications that require the new tokens will then have their versions of the pie libs updated via another PR. Once this PR is merged, the next version of the application to be released will contain the design token updates.
```

Comment:
Should this be `PIE libraries`?

Reply:
@jordifornellsJET please clarify on this one :)

Reply:
Hey! So I didn't write these bits and am also not sure what it means or should be.